akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -81,11 +82,13 @@ are executed sequentially after successful execution of the persist statement (o
 
 When an event has been persisted successfully the new state is created by applying the event to the current state with the `eventHandler`.
 
-The event handler returns the new state, which must be immutable so you return a new instance of the state.
+The state is typically immutable and then the event handler returns a new instance of the state. If the state is
```

Comment:
I'd clarify this. If it is immutable 
```suggestion
The state is immutable, the event handler returns a new instance of the state. If the state is
```
And clarify when or why state is not immutable.